src/Bloc-Alexandrie/BlBorder.extension.st

```diff
@@ -20,14 +19,12 @@ BlBorder >> aeApplyTo: aeCanvas element: aBlElement [
 	aeCanvas setBorderBlock: [
 		paint aeApplyTo: aeCanvas.
 
-		aeCanvas
-			setDashes: style dashArray
-			offset: style dashOffset.
+		"No dashes is the default"
+		style dashArray = aBlElement parent border style dashArray ifFalse: [
+			aeCanvas setDashes: style dashArray offset: style dashOffset ].
 
 		aBlElement geometry hasJoins ifTrue: [
-			style lineJoin
-				aeApplyTo: aeCanvas
-				style: style ].
+			style lineJoin aeApplyTo: aeCanvas style: style ].
```

Comment:
This is not related to the drag and drop examples

Reply:
No I agree, I just don't know why iceberg decided to change this file although it's only formatting the message on a single line

Reply:
But not only format, it also adds the `ifFalse:`. Mmm what's the most practical way to remove this diff. I'll check

Reply:
ah yes indeed sorry I didn't see it in the embed comment but yes these are changes that were previously in Bloc and my image was older than the time you changed it, I tried to pull the most recent commits and not make these appear in my PR but it appears I still have some things to learn about git and how to deal with this kind of issue

Reply:
actually, can I put my changes on another branch of my repo, delete the changes on this BlBorder class and do another PR ? I think we'd lose the history of my commits but it's not **that** important

Reply:
Hi Martin, can you help me on this pls ?

Reply:
hi @Enzo-Demeulenaere yes let me check!